docs/core/testing/unit-testing-platform-extensions-code-coverage.md

```diff
@@ -30,12 +30,21 @@ Microsoft Code Coverage provides the following options:
 | Option                     | Description                                                                   |
 |----------------------------|-------------------------------------------------------------------------------|
 | `--coverage`               | Collect the code coverage using dotnet-coverage tool.                         |
-| `--coverage-output`        | Output file.                                                                  |
-| `--coverage-output-format` | Output file format. Supported values are: 'coverage', 'xml', and 'cobertura'. |
-| `--coverage-settings`      | XML code coverage settings.                                                   |
+| `--coverage-output`        | The name or path of the produced coverage file. By default, the file is `TestResults/<guid>.coverage`. |
+| `--coverage-output-format` | Output file format. Supported values are: `coverage`, `xml`, and `cobertura`. Default is `coverage`. |
+| `--coverage-settings`      | [XML code coverage settings](https://learn.microsoft.com/dotnet/core/additional-tools/dotnet-coverage#settings). |
 
 For more information about the available options, see [settings](../additional-tools/dotnet-coverage.md#settings) and [samples](https://github.com/microsoft/codecoverage/tree/main/samples/Algorithms).
 
 ## Coverlet
 
+> [!IMPORTANT]
+> The `coverlet.collector` NuGet package is designed specifically for VSTest and cannot be used with `Microsoft.Testing.Platform`
+
 There's currently no Coverlet extension, but you can use [Coverlet .NET global tool](https://github.com/coverlet-coverage/coverlet#net-global-tool-guide-suffers-from-possible-known-issue).
+
+Assuming you have already installed coverlet global tool, you can now run:
+
+```bash
+coverlet .\bin\Debug\net8.0\TestProject2.dll --target "dotnet" --targetargs "test .\bin\Debug\net8.0\TestProject2.dll --no-build"
```

Comment:
Is it expected to pass the dll instead of exe to `dotnet test`?

Reply:
Both should work